### ISSUE TITLE
CPU

### ISSUE THREAD

**Issue by meng0li** [jwyang/faster-rcnn.pytorch#772]
Hi, can the code run in win10+pytorch(CPU) environment？

**Comment by loolzaaa** [comment]
This version works on CPU only on old PyTorch 0.4 version and python 2.x

If you work with PyTorch 1.x and python 3.x you can try my [my implementation of this work](https://github.com/loolzaaa/faster-rcnn-pytorch) that full work on CPU.

**Comment by meng0li** [comment]
Hello, I took a look at the link you provided, may I ask if there is a detailed readme tutorial？





------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"loolzaaa"<notifications@github.com&gt;;
发送时间:&nbsp;2020年4月14日(星期二) 下午2:52
收件人:&nbsp;"jwyang/faster-rcnn.pytorch"<faster-rcnn.pytorch@noreply.github.com&gt;;
抄送:&nbsp;"痴"<2957574745@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [jwyang/faster-rcnn.pytorch] CPU (#772)





 
This version works on CPU only on old PyTorch 0.4 version and python 2.x
 
If you work with PyTorch 1.x and python 3.x you can try my my implementation of this work that full work on CPU.

**Comment by loolzaaa** [comment]
I'm working on it right now. Coming soon.

**Comment by meng0li** [comment]
Do you need to change the code to run in a CPU environment?




---Original---

**Comment by loolzaaa** [comment]
> Do you need to change the code to run in a CPU environment?

No, network implementation that i left by link above already working on CPU. For quick start, just follow readme.

Now i'm working on some docs in addition to help message in CLI.
